library/tasks/CorrectUmiMarkDuplicates.wdl

```diff
@@ -1,20 +1,41 @@
-task CorrectUmiMarkDuplicates {
+task SortAndCorrectUmiMarkDuplicates {
   File bam_input
+
+  # runtime values
+  String docker = "quay.io/humancellatlas/secondary-analysis-picard:v0.2.2-2.10.10"
+  Int machine_mem_mb = 7500
+  # give the command 500MB of overhead
+  Int command_mem_mb = machine_mem_mb - 500
+  Int cpu = 1
   # mark duplicates swaps a large amount of data to disk, has high disk requirements.
-  Int estimated_required_disk = ceil(size(bam_input, "G") * 6)
+  Int disk = ceil(size(bam_input, "G") * 6)
+  Int preemptible = 0
+
+  meta {
+    description: "AMBROSE HALP!!"
+  }
+
+  parameter_meta {
+    bam_input: ""
```

Comment:
This parameter is pretty self-explanatory, but I generally still prefer having a description. How abut `The input bam file for this task`?